testsuite/MDAnalysisTests/coordinates/test_chainreader.py

```diff
@@ -206,6 +208,12 @@ def test_set_all_format_tuples(self):
         assert universe.trajectory.n_frames == 21
         assert_equal(universe.trajectory.filenames, [PDB, XTC, TRR])
 
+    def test_set_all_format_lammps(self):
```

Comment:
Could you also add a test that the time is as expected.

Reply:
What do you think of this added test?

Reply:
yeah looks great!